add mdadm support

SourceDoctor opened the pull request:
snmp extend for mdadm Monitoring:
`extend mdadm /etc/snmp/mdadm`


**mdadm.conf**
```
CAT=/bin/cat
LS=/bin/ls
```

VVelox commented:
Nice. I am curious commits in LibreNMS for this?

That said starting work on RAID monitoring here. Wrote the poller for it here... https://github.com/VVelox/Device-RAID-Poller . It is is own repo given the complexity of it.

SourceDoctor commented:
i changed the code, so a configfile is optional.

@VVelox  i didn't find something like "I'm working on a raid or mdadm monitoring enhancement", so after a view month waiting i started to write it myself.

On librenms project i'm also waiting to get pull request accepted. 
I changed like requested communication from from plain snmp to json.
https://github.com/librenms/librenms/pull/10159

VVelox commented:
@SourceDoctor Cool. Getting ready to add in support for generalized RAID monitoring soon as well. :)
